specs/commandline/inspect.md

```diff
@@ -0,0 +1,204 @@
+
+# notation inspect
+
+## Description
+
+Use `notation inspect` command to inspect all the signatures associated with signed artifact in a human readable format.
+
+Upon successful execution, both the digest of the signed artifact and the digests of signatures manifest along with their properties associated with signed artifact are printed in the following format:
+
+```shell
+<registry>/<repository>@<digest>
+├── application/vnd.cncf.notary.signature
+  ├──<digest_of_signature_manifest>
+    ├──<signed attributes>
+    ├──<user-defined attributes>
+    ├──<unsigned Attributes>
+    ├──<certificates>
+    ├──<payload>
+  ├──<digest_of_signature_manifest>
+    ├──<signed attributes>
+    ├──<unsigned attributes>
+    ├──<certificates>
+    ├──<payload>
+```
+
+## Outline
+
+```text
+Inspect all signatures with the signed artifact.
+
+Usage:
+    notation inspect [flags] <reference>
+    notation inspect <registry>/<repository>:<tag>
+
+Flags:
+   -h, --help for describing the signature
+   -o, --output on command line sets the output to json
+   -p, --password string password for registry operations (default to $NOTATION_PASSWORD if not specified)
+       --plain-http registry access via plain HTTP
+   -u, --username string username for registry operations (default to $NOTATION_USERNAME if not specified)
+```
+
+## Usage
+
+### Display the details of all the listed signatures and its associated certificate properties of the signed container image
+
+
+```text
+notation inspect <registry>/<repository>@<digest> [flags] <reference>
+```
+
+## Inspect signatures on the supplied OCI artifact identified by the digest
+
+$ notation inspect localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
+
+An example output:
+```shell
+localhost:5000/net-monitor@sha256:b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9
+├──application/vnd.cncf.notary.signature
+    ├── sha256:aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
+      ├──signed attributes:
+            ├──contentType: application/vnd.cncf.notary.payload.v1+json
+            ├──signingScheme: notary.default.x509
```

Comment:
Should we add a new attribute [Signature Algorithm](https://github.com/notaryproject/notaryproject/blob/main/specs/signature-specification.md#signature-algorithm-requirements) here?